ouroboros-consensus-test/src/Test/LedgerTables.hs

```diff
@@ -0,0 +1,51 @@
+{-# LANGUAGE FlexibleContexts #-}
+
+module Test.LedgerTables (
+    prop_stowable_laws
+  , prop_tablestuff_laws
+  ) where
+
+import           Data.Function (on)
+
+import           Ouroboros.Consensus.Ledger.Basics
+import           Ouroboros.Consensus.Ledger.Tables
+
+import           Test.QuickCheck
+
+compareTables ::
+     ( IsMapKind mk
+     , TableStuff (LedgerState blk)
+     , Eq (LedgerTables (LedgerState blk) mk)
+     , Show (LedgerTables (LedgerState blk) mk)
+     )
+  => LedgerState blk mk
+  -> LedgerState blk mk
+  -> Property
+compareTables = (===) `on` projectLedgerTables
+
+prop_stowable_laws ::
+     ( TableStuff (LedgerState blk)
+     , StowableLedgerTables (LedgerState blk)
+     , Eq (LedgerTables (LedgerState blk) EmptyMK)
+     , Show (LedgerTables (LedgerState blk) EmptyMK)
+     , Show (LedgerTables (LedgerState blk) ValuesMK)
+     )
+  => LedgerState blk EmptyMK
+  -> Property
+prop_stowable_laws = \ls ->
+  compareTables (stowLedgerTables (unstowLedgerTables ls)) ls .&&.
+  let ls' = unstowLedgerTables ls
+  in compareTables (unstowLedgerTables (stowLedgerTables ls')) ls'
+
+prop_tablestuff_laws ::
```

Comment:
Maybe add a comment to these properties that describes the laws, since it is a little hard to read from the code itself what the laws are that are being tested. Something like these crude examples from the issue title: `with . project = id` and `unstow . stow= id`

Reply:
Added